src/main/java/pages/WishListPage.java

```diff
@@ -0,0 +1,54 @@
+package pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import pages.section.Header;
+
+import java.util.List;
+
+import static util.TestUtils.isDisplayed;
+
+public class WishListPage extends BasePage {
+
+    public WishListPage() {
```

Comment:
move the constructor to place after the class fields

Reply:
Fixed